src/main/java/org/opentripplanner/middleware/otp/OtpVersion.java

```diff
@@ -25,4 +30,23 @@ public String uri() {
         return uri;
     }
     public String toString() { return name; }
+
+    public static OtpVersion intToVersion(int versionAsInt) {
```

Comment:
Change the method to `private`. Consider changing this method name to `getOtpVersion` and the parameter to `int version`.